src/utils/validations.js

```diff
@@ -112,19 +112,15 @@ const validateTripRequest = async (req, res, next) => {
   }
   const whiteSpaces = /\s+/g;
   travelType = String(travelType).replace(whiteSpaces, ' ').trim().toLowerCase();
-  switch (travelType) {
-    case 'one-way':
-      try {
-        const validationOutput = await validateOneWayTripRequest(tripRequestInfo);
-        validationOutput.userId = req.sessionUser.id;
-        req.body = validationOutput;
-        return next();
-      } catch (validationError) {
-        return errorResponse(res, badRequest, validationError.message);
-      }
-    default:
-      return errorResponse(res, badRequest, invalidTravelType);
-  }
+    try {
+      const validationOutput = travelType === 'one-way' ? await validateOneWayTripRequest(tripRequestInfo)
```

Comment:
This way you used here, will be difficult when there will come to more than to `tripTypes`, as @karamuka said, you should have tried tp use switch case